docs/source/posts/2024/2024-06-06-week1-wachiou-bouraima.rst

```diff
@@ -0,0 +1,61 @@
+WEEK 1: Progress and challenges at Google Summer of Code (GSoC) 2024
+====================================================================
+
+.. post:: June 06, 2024
+   :author: Wachiou BOURAIMA
+   :tags: google
+   :category: gsoc
+
+Hello👋🏾,
+
+Welcome back to my Google Summer of Code (GSoC) 2024 journey!
+This week has been filled with progress and challenges as I continue to work on modernizing the FURY code base.
+
+
+Applying the keyword_only decorator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+My main task this week was to apply the keyword_only decorator to several functions.
+The decorator ensures that all arguments except the first are keyword-only,
```

Comment:
Why do you split these lines so short?

Reply:
@WassCodeur ?

Reply:
Sorry @itellaetxe 

I thought I'd answered that question. 

My computer screen isn't too big, so to avoid always scrolling to see the next part of the line, which is too long, I try to split the line, so I can read it easily on my screen.

Reply:
Okay I get it now, thanks Wachiou. I should have figured it out on my own tbh